mimic/test/test_util.py

```diff
@@ -83,3 +83,9 @@ def test_unicode_body(self):
         self.failureResultOf(
             request(self, Resource(), b"POST", b"", u"not bytes")
         )
+
+    def test_bad_request(self):
```

Comment:
This test should have a docstring explaining what it's testing.  I am not quite clear on the purpose of `bad_request` - is this the proper format of the actual response returned by Nova when selecting an invalid disk config?  (You wouldn't happen to have a documentation reference there, would you?  It would be great to link that from the test.)